subsystem/image_test.go

```diff
@@ -123,113 +101,6 @@ var _ = Describe("system-test image tests", func() {
-		cluster, err = userBMClient.Installer.RegisterCluster(ctx, &installer.RegisterClusterParams{
-			NewClusterParams: &models.ClusterCreateParams{
-				Name:             swag.String("test-cluster"),
-				OpenshiftVersion: swag.String(openshiftVersion),
-				PullSecret:       swag.String(pullSecret),
-			},
-		})
-		Expect(err).NotTo(HaveOccurred())
-		clusterID = *cluster.GetPayload().ID
-	})
-
-	It("generate_image_after_proxy_was_set", func() {
-		// Generate ISO of registered cluster without proxy configured
-		_, err := userBMClient.Installer.GenerateClusterISO(ctx, &installer.GenerateClusterISOParams{
-			ClusterID:         clusterID,
-			ImageCreateParams: &models.ImageCreateParams{},
-		})
-		Expect(err).NotTo(HaveOccurred())
-
-		// Update cluster with proxy settings
-		httpProxy := "http://proxyserver:3128"
-		noProxy := "test.com"
-		_, err = userBMClient.Installer.UpdateCluster(ctx, &installer.UpdateClusterParams{
-			ClusterUpdateParams: &models.ClusterUpdateParams{
-				HTTPProxy: &httpProxy,
-				NoProxy:   &noProxy,
-			},
-			ClusterID: clusterID,
-		})
-		Expect(err).ShouldNot(HaveOccurred())
-
-		// Verify proxy settings changed event emitted
-		verifyEventExistence(clusterID, "Proxy settings changed")
-
-		// at least 10s must elapse between requests to generate the same ISO
-		time.Sleep(time.Second * 10)
-
-		// Generate ISO of registered cluster with proxy configured
-		_, err = userBMClient.Installer.GenerateClusterISO(ctx, &installer.GenerateClusterISOParams{
-			ClusterID:         clusterID,
-			ImageCreateParams: &models.ImageCreateParams{},
-		})
-		Expect(err).NotTo(HaveOccurred())
-	})
-
-	It("[V2UpdateCluster] generate_image_after_proxy_was_set", func() {
-		// Generate ISO of registered cluster without proxy configured
-		_, err := userBMClient.Installer.GenerateClusterISO(ctx, &installer.GenerateClusterISOParams{
-			ClusterID:         clusterID,
-			ImageCreateParams: &models.ImageCreateParams{},
-		})
-		Expect(err).NotTo(HaveOccurred())
-
-		// Update cluster with proxy settings
-		httpProxy := "http://proxyserver:3128"
-		noProxy := "test.com"
-		_, err = userBMClient.Installer.V2UpdateCluster(ctx, &installer.V2UpdateClusterParams{
-			ClusterUpdateParams: &models.V2ClusterUpdateParams{
-				HTTPProxy: &httpProxy,
-				NoProxy:   &noProxy,
-			},
-			ClusterID: clusterID,
-		})
-		Expect(err).ShouldNot(HaveOccurred())
-
-		// Verify proxy settings changed event emitted
-		verifyEventExistence(clusterID, "Proxy settings changed")
-
-		// at least 10s must elapse between requests to generate the same ISO
-		time.Sleep(time.Second * 10)
-
-		// Generate ISO of registered cluster with proxy configured
-		_, err = userBMClient.Installer.GenerateClusterISO(ctx, &installer.GenerateClusterISOParams{
-			ClusterID:         clusterID,
-			ImageCreateParams: &models.ImageCreateParams{},
-		})
-		Expect(err).NotTo(HaveOccurred())
-	})
-
-})
-
-func verifyEventExistence(ClusterID strfmt.UUID, message string) {
```

Comment:
from commit-msg:

> The verifyEventExistence helper function is moved to the metrics_test
file so that it will be co-located with the last remaining caller.

Do we remove it? I don't see it in a new location.

Reply:
Ah yeah, I ended up removing it because we didn't actually need to create an image in any of these tests. I'll update the commit.